src/interactors/AuthDevice.js

```diff
@@ -0,0 +1,17 @@
+class AuthenticateDevice {
+  constructor(cloud, queue) {
+    this.cloud = cloud;
+    this.queue = queue;
+  }
+
+  async execute(id, token) {
+    const status = await this.cloud.authDevice(id, token);
+    if (status === 403) {
+      return this.queue.send('fog', 'device.auth', { id, status });
+    }
+
+    return this.queue.send('fog', 'device.auth', { id, status });
```

Comment:
Do we really need to replicate the HTTP status code to amqp?
Shouldn't be better: `{id: ..., authenticated: true/false}`

Reply:
It's an option since the status code is well-known, but I want to choose what is better for the underlying services to use. Do you think this response will help in some way when verifying it on the KNoT service (knotd)?

Reply:
I really don't know which is the better option.
Now the knotd doesn't handle the error code. But this can be a good point if we map this error code to DBus error and it is shown on WebUI.